### PR TITLE
Gaea machine name

### PR DESCRIPTION
**Pull request by zach1221**
## Commit Queue Requirements:

- [ ] Fill out all sections of this template.
- [ ] All sub component pull requests have been reviewed by their code managers.
- [ ] Run the full RT suite (compared to current baselines) on either Hera/Derecho/Hercules AND have committed the log to my PR branch.
- [ ] Add list of all failed regression tests in "Regression Tests" section.

### Priority
- [ ] Critical Bugfix (This PR contains a critical bug fix and should be prioritized.)
- [ ] High (This PR contains a feature or fix needed for a time-sensitive project (eg, retrospectives, implementations))
- [ ] Normal

### Subcomponent (with links)

- [ ] AQM
- [ ] CDEPS
- [ ] CICE
- [ ] CMEPS
- [ ] CMakeModules
- [ ] FV3
- [ ] GOCART
- [ ] HYCOM
- [ ] MOM6
- [ ] NOAHMP
- [ ] WW3
- [ ] stochastic_physics
- [ ] none

### Input data
- [ ] No changes are expected to input data.
- [ ] Changes are expected to input data:
  - [ ] New input data.
  - [ ] Updated input data.

### Regression Tests:
- [ ] No changes are expected to any regression test.
- [ ] Changes are expected to the following tests:
<details><summary>FAILED REGRESSION TESTS</summary>

</details>

### Libraries

- [ ] Not Needed
- [ ] Needed
  - [ ] Create separate issue in [JCSDA/spack-stack](https://github.com/JCSDA/spack-stack) asking for update to library. Include library name, library version.
  - [ ] Add issue link from JCSDA/spack-stack following this item 

### Testing Log:
- RDHPCS
  - [ ] Hera
  - [ ] Orion
  - [ ] Hercules
  - [ ] Jet
  - [ ] Gaea
  - [ ] Derecho
- WCOSS2
  - [ ] Dogwood/Cactus
  - [ ] Acorn
- CI
  - [ ] Completed
- opnReqTest
  - [ ] N/A
  - [ ] Log attached to comment

### CODE REVIEW

**Comment by ulmononian** [comment]
@zach1221 most of these changes look good, but some seem redundant to commits already in my pr branch. a can you check if your updates reflect the most up to date version of my branch?

File: tests/compile.sh [hunk 1]
```diff
@@ -58,8 +58,8 @@ if [[ $MACHINE_ID == macosx ]] || [[ $MACHINE_ID == linux ]]; then
   source $PATHTR/modulefiles/ufs_${MACHINE_ID}.${RT_COMPILER}
 else
   # Activate lua environment for gaea c5
-  if [[ $MACHINE_ID == gaea-c5 ]]; then
-    source /lustre/f2/dev/role.epic/contrib/Lmod_init_C5.sh
+  if [[ $MACHINE_ID == gaea ]]; then
+    source /gpfs/f5/epic/scratch/role.epic/contrib/Lmod_init_C5.sh
```
Comment (ulmononian) [inline tests/compile.sh]: @natalie-perlin is the `source /gpfs/f5/epic/scratch/role.epic/contrib/Lmod_init_C5.sh` still needed here? i know you mentioned that we do not need to source this file in `module-setup.sh` anymore, and i've replaced it there with `module reset`. thanks!

File: tests/module-setup.sh [hunk 2]
```diff
@@ -75,7 +75,7 @@ elif [[ $MACHINE_ID = gaea-c5 ]] ; then
         # /etc/profile here.
         source /etc/profile
     fi
-    source /lustre/f2/dev/role.epic/contrib/Lmod_init_C5.sh
+    source /gpfs/f5/epic/scratch/role.epic/contrib/Lmod_init_C5.sh
```
Comment (ulmononian) [inline tests/module-setup.sh]: @zach1221 @natalie-perlin my understanding is that we just need to do `module reset` here now.

File: tests/rt.sh [hunk 3]
```diff
@@ -304,30 +304,32 @@ elif [[ $MACHINE_ID = acorn ]]; then
...
-  module use /lustre/f2/dev/role.epic/contrib/C5/rocoto/modulefiles
+  module use /ncrc/proj/epic/rocoto/modulefiles
   module load rocoto
   ROCOTORUN=$(which rocotorun)
   ROCOTOSTAT=$(which rocotostat)
   ROCOTOCOMPLETE=$(which rocotocomplete)
   ROCOTO_SCHEDULER=slurm
 
+  module use /ncrc/proj/epic/miniconda3/modulefiles
+  module load miniconda3/4.12.0
```
Comment (ulmononian) [inline tests/rt.sh]: i don't think we need to load miniconda here

Comment (ulmononian) [reply tests/rt.sh]: do we? python is loaded below which is sufficient for spack-stack

File: tests/compile.sh (hunk 1 above)
Comment (zach1221) [reply tests/compile.sh]: @ulmononian I've changed mine to module reset to match yours.

File: tests/module-setup.sh (hunk 2 above)
Comment (zach1221) [reply tests/module-setup.sh]: @ulmononian  good here

File: tests/rt.sh (hunk 3 above)
Comment (zach1221) [reply tests/rt.sh]: @ulmononian done.

File: tests/rt.sh [hunk 4]
```diff
@@ -304,30 +304,29 @@ elif [[ $MACHINE_ID = acorn ]]; then
...
   module load rocoto
   ROCOTORUN=$(which rocotorun)
   ROCOTOSTAT=$(which rocotostat)
   ROCOTOCOMPLETE=$(which rocotocomplete)
   ROCOTO_SCHEDULER=slurm
 
+  
   module load PrgEnv-intel/8.3.3
   module load intel-classic/2023.1.0
-  module load cray-mpich/8.1.25
```
Comment (ulmononian) [inline tests/rt.sh:318]: can you please add `module load cray-mpich/8.1.25` back?

Comment (zach1221) [reply tests/rt.sh:318]: @ulmononian ok module added back.

**Review by ulmononian — APPROVED** [review]

**Pull request merged**